m3-sys/llvm/llvm9/src/Main.m3

```diff
@@ -288,6 +288,11 @@ VAR
     GCharDataRep := "e-m:e-p:32:32-i32:32-i64:32:64-f80:128-n8:16:32:64-S128";
 *)
 
+(* windows x64 a-ka windows x86_64
+    GCharTriple := "x86_64-pc-windows-msvc";
+    GCharDataRep := "e-m:w-i64:64-f80:128-n8:16:32:64-S128";
+*)
```

Comment:
Please call it "amd64".
Is "f80" correct?

Reply:
1) Fixed in https://github.com/VictorMiasnikov/cm3/commit/40c8521812e6f17f62965e65351946a400c4970b
2)

} Is "f80" correct?

I think yes, it's correct:  always "64-f80"
==
(* amd64 *)
    GCharTriple := "x86_64-pc-linux-gnu";
    GCharDataRep := "e-m:e-p:64:64-i64:64-f80:128-n8:16:32:64-S128";

(* linuxlibc6 
    GCharTriple := "i686-pc-linux-gnu";
    GCharDataRep := "e-m:e-p:32:32-i32:32-i64:32:64-f80:128-n8:16:32:64-S128";
*)

(* windows amd64
    GCharTriple := "x86_64-pc-windows-msvc";
    GCharDataRep := "e-m:w-i64:64-f80:128-n8:16:32:64-S128";
*)

(* windows 686  
    GCharTriple := "i686-pc-windows-msvc";
    GCharDataRep := "e-m:x-p:32:32-i64:64-f80:32-n8:16:32-a:0:32-S32";
*)
==